client/src/Main.elm

```diff
@@ -16,10 +17,11 @@ import Element.Region
 import File exposing (File)
 import File.Select as Select
 import Html exposing (Html, a)
-import Html.Attributes as HA
+import Html.Attributes as HA exposing (required)
```

Comment:
Why not `HA.required`?

Reply:
This was a suggestion from the review test.
I can't find any occurence of HA. required.